app/manager-frontend/src/components/ActivitySourceModal.jsx

```diff
@@ -0,0 +1,38 @@
+import {
+  Button,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+} from "@chakra-ui/react";
+
+export const ActivitySourceModal = ({
```

Comment:
```suggestion
export const DeleteActivitySourceModal = ({
```

🔧 can we rename the file to match too?